docs/faq.md

```diff
@@ -80,3 +81,28 @@ Currently, the main benefit of [signed releases](/checks.md#signed-releases) is
 However, there are already moves to make it even more relevant. For example, the OpenSSF is working on [implementing signature verification for NPM packages](https://github.blog/2022-08-08-new-request-for-comments-on-improving-npm-security-with-sigstore-is-now-open/) which would allow a consumer to automatically verify if the package they are downloading was generated through a reliable builder and if it is correctly signed.
 
 Signing releases already has some relevance and it will soon offer even more security benefits for both consumers and maintainers.
+
+### Branch-Protection: How to setup a 10/10 branch protection on GitHub?
+
+To get a 10/10 score for Branch-Protection check using a non-admin token, you should have the following settings for your branches:
```

Comment:
I think these paragraph with the images would fit better *after* the "It's important to reiterate [...]" paragraph and the table, in a form of example/conclusion.

Reply:
I rather leave the current structure because the important question I wanna answer is how to setup the branch protection settings, so should be the first information possible. If the user is not getting a perfect score, then they would look at the tier based score table to understand why.

Reply:
Hmm I see your point. Yep, makes sense